common/src/main/java/com/revenuecat/purchases/common/errorLogUtils.kt

```diff
@@ -0,0 +1,101 @@
+package com.revenuecat.purchases.common
+
+import com.revenuecat.purchases.strings.Emojis
+
+fun googleErrorLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.GOOGLE_ERROR, Emojis.DOUBLE_EXCLAMATION)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
```

Comment:
This when statement is repeated in all methods, I would extract it in a private method so it's not accessible from outside this file. I would probably call it `logMessageAccordingToLevel(LogLevel, String)` or something like that.

Or maybe we could rename the `log` function in `logUtils` to `infoLog` (since it's always Log.w) and make a `log` function that accepts a `LogLevel` and that the only thing it does is what you are doing with this `when`. `LogLevel` would have to be extracted from this file. What do you think @aboedo

Reply:
agreed. I'd skew towards just wrapping it in `logUtils` so this class doesn't even have to know about it.

Reply:
it seems like all the methods in this file just do the same things with different params, so they should be calling a single method with the different params (so not just unifying the `when`, but also the emojis bit)